### DISCUSSION TITLE
Irrelevant Intro in React foundations chapter 7 in new docs

### DISCUSSION THREAD

**Discussion by pydi99** [vercel/next.js#57589] [App Router]
@leerob hey there I have been going through the new next.js learn docs, and in chapter 7 of  react foundations [https://nextjs.org/learn/react-foundations/updating-state](url). I think there is something irrelevant.
At the start of the chapter where it says 
```
In this chapter...
Here are the topics we’ll cover
Create the dashboard pages using file-system routing.......
```
and the quiz part is irrelevant to the topic that was discussed.
Kindly update it please.

**Comment by Mriganka5137** [comment]
Yeah. I found this too🥸

**Reply by pydi99** [reply]
did you gave feedback? I was told that they are gonna read feedbacks and update accordingly.

**Reply by Mriganka5137** [reply]
Not yet.
